packages/tooltips/src/containers/TooltipContainer.js

```diff
@@ -90,13 +92,18 @@ class TooltipContainer extends ControlledComponent {
      * @param {Function} renderProps.getTriggerProps - Props to be spread onto the trigger element
      * @param {Function} renderProps.isVisible - Whether the Tooltip is currently visible
      */
-    trigger: PropTypes.func
+    trigger: PropTypes.func,
+    /**
+     * The z-index of the popper.js placement container
+     */
+    zIndex: PropTypes.oneOfType([PropTypes.number, PropTypes.oneOf(['auto', 'inherit', 'unset'])])
```

Comment:
Missing `initial` – but I'm not sure we get too fussy about specifying props to this level of detail. We may prefer a simpler `PropTypes.oneOfType([PropTypes.number, PropTypes.string])`. But I'll let @austin94 weigh-in to say for sure.

Reply:
I'll change this after Austin weighs in. Thanks for taking a look!

Reply:
Yeah no need to specify exact string values `string` is enough